docs/docs/awesome-gatsby-resources.md

```diff
@@ -81,6 +81,7 @@ See the [library of official and community plugins](/plugins/)
 - [Understanding the Gatsby lifecycle](https://medium.com/narative/understanding-gatsbys-lifecycle-31c473ba2f2d)
 - [GatsbyGuides.com series on building web applications with Gatsby & Drupal](https://gatsbyguides.com/)
 - [Deep dive into Gatsby – Building a static blog using Gatsby, React and GraphQL](https://zocada.com/deep-dive-gatsby-building-a-simple-static-blog-using-gatsby-react-and-graphql/)
+- [Gatsby on AWS Lambda](https://gist.github.com/digitalkaoz/94933c246ba67032a1507083e2605a30)
```

Comment:
It would help to link to an article about the code example, as a gist alone would potentially be difficult for Gatsby learners to follow. As such, we'd prefer not to link to raw source code.

Reply:
@marcysutton i've only link to code Sorry